parsons/google/google_bigquery.py

```diff
@@ -349,6 +350,75 @@ def query_with_transaction(self, queries, parameters=None):
         """
         self.query(sql=queries_wrapped, parameters=parameters, return_values=False)
 
+    def export_table_to_gcs(
+        self,
+        table_name: str,
+        destination_uris: str,
+        location: str = "US",
+        destination_file_format: str = "CSV",
+        field_delimiter: str = ",",
+        compression: str = None,
+        job_config: dict = {},
+        **export_kwargs,
+    ):
+        """
+        Export a bigquery table to GCS
+
+        `Args:`
+            table_name: str
+                The table name to export.
+            destination_uris: str
+                Uri indicating where the table will be exported to in GCS
```

Comment:
sometimes a wildcard is required if a file is big right? we should put that in the docs